remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -100,6 +102,9 @@
     private static final Category CATEGORY = createDefaultCategory();
     private static final String CONTENT_NODE_ID = "content-node-id";
     private static final NodeRef CONTENT_NODE_REF = createNodeRefWithId(CONTENT_NODE_ID);
+    private static final String MOCK_ROOT_LEVEL = "/{mockRootLevel}";
+    private static final String MOCK_CHILD_LEVEL = "/{mockChild}";
+    private static final String MOCK_CATEGORY_PATH = "//" + MOCK_ROOT_LEVEL + "//" + MOCK_CHILD_LEVEL;
```

Comment:
I don't understand all the slashes in this line. It ends up as `///{mockRootLevel}///{mockChild}`

Presumably it's unimportant what the exact string is.